week2/homework/src/index.js

```diff
@@ -1,3 +1,70 @@
 'use strict';
 
 // TODO: Write the homework code in this file
+
+const fs = require('fs');
+
+let fileName = 'toDo.json';
+// console.log(fileName);
+function readFile() {
+    return new Promise(
+        resolve => fs.readFile(
+            fileName,
+            (err, data) => resolve(err ? '' : data.toString())
+        )
+    );
+}
+
+function remove() {
+    readFile()
+        .then(content => {
+            let newArray = content.split('\n');
+            let i = newArray.length-1
+            newArray.splice(i, 1);
+           let new2 =  newArray.join('\n')    
+            writeFile(new2);
+            console.log(content);
+        })
+}
+
+function writeFile(...text) {
+    return new Promise(
+        (resolve, reject) => fs.appendFile(
+            fileName,
+            `${text.join(' ')}\n`,
+            (err, data) => err
+                ? reject(err)
+                : resolve(data)
+        )
+    );
+}
+
+const cmd = process.argv[2];
+const args = process.argv.slice(3);
+const removeArgs = args[0];
+
+switch (cmd) {
+  case "add":
+    writeFile(...args)
```

Comment:
By using the rest operator `...` here and in your function definition, you're essentially just passing the `args` array. You could simply do `writeFile(args)` and then define your `writeFile` function like so: `function writeFile(text)`. Does that make sense?